src/Tizen.NUI/src/internal/WebView/WebView.cs

```diff
@@ -181,6 +186,27 @@ protected override void Dispose(DisposeTypes type)
         [EditorBrowsable(EditorBrowsableState.Never)]
         public delegate void JavaScriptPromptCallback(string message1, string message2);
 
+        /// <summary>
+        /// The callback function that is invoked when screen shot is captured.
+        /// </summary>
+        [UnmanagedFunctionPointer(CallingConvention.StdCall)]
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public delegate void ScreenshotCapturedCallback(ImageView image);
```

Comment:
I wonder why they are not events.

Reply:
This callback is called when getting screen shot asynchronously:
public bool GetScreenshotAsynchronously(Rectangle viewArea, float scaleFactor, ScreenshotAcquiredCallback callback)
you mean,  ScreenshotCapturedCallback is renamed to ScreenshotCapturedEventHandler?
but if that, there would be no difference between callback and real event hanlder, such as PageLoadErrorEventHandler.

Reply:
The events we already provide, such as, PageLoadFinished is used like,
```C#
webView.PageLoadeFinished += OnPageLoadFinished;
webView.LoadUrl(url);
// OnPageLoadFinished is called when the page is loaded.
```

So I though the new callbacks have no different with PageLoadFinished.
```C#
webView.ScreenShotAcquired += OnScreenShotAcquired;
webView.TakeScreenshot(viewArea, scaleFactor);
```

Reply:
All is ok.
then RegisterJavaScriptAlertCallback/RegisterJavaScriptConfirmCallback/RegisterJavaScriptPromptCallback need be changed too?

Reply:
Sorry, I checked and have another question.
If RegisterGeolocationPermissionCallback  is changed to event,
many event handlers can be added.
but RegisterGeolocationPermissionCallback had better be called only once.

Reply:
If so, when user call `RegisterGeolocationPermissionCallback` multiple times, only last one registered?

Reply:
yes, only last one is valid.

Reply:
OK then I will go with your current code :)